libsofia-sip-ua/msg/msg_date.c

```diff
@@ -248,7 +248,8 @@ issize_t msg_date_d(char const **ss, msg_time_t *date)
     /* actime-date =
        wkday SP month SP ( 2DIGIT | ( SP 1DIGIT )) SP time SP 4DIGIT */
     mon = month_d(s); skip_token(&s);
-    if (mon < 0 || !IS_LWS(*s)) return -1; s++;
+    if (mon < 0 || !IS_LWS(*s)) return -1;
+    s++;
```

Comment:
This one is interesting- I assume the `s++` is correct and the compiler didn't uncover a bug?

Reply:
Looks OK- the `while (IS_LWS(*s)) s++;` below suggests this is correct and was simply confusing whitespace.